### PR TITLE
fix: issue video music upload

### LINKED ISSUE #1053: Mobile: Upload music
Can’t dismiss keyboard 
Can’t file picker (only iPhone test)

Maybe remove if broken. Re add when fixed?

IMG_2530

### LINKED ISSUE #1055: Mobile: Video upload can’t upload
Screenshot 
Layout issues
Not working 
Can’t dismiss keyboard

### PR DESCRIPTION
**Pull request by sachin-into**
Issues Fixed:
-fixes #1055 
-fixes #1053

### CODE REVIEW

**Review by omniwired — APPROVED** [review]
code is good, I didn't test it on a real device.

File: packages/components/mini/SelectPicture.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            )}
+          </View>
+        </View>
+      ) : (
+        <SVG
+          source={cameraSVG}
+          height={24}
+          width={24}
+          style={{
+            opacity: Array.isArray(files) && files?.length > 0 ? 0.7 : 1,
```
Comment (omniwired) [inline packages/components/mini/SelectPicture.tsx:122]: ✨

**Review by omniwired — DISMISSED** [review]

**Comment by n0izn0iz** [comment]
fix conflicts pls

**Review by n0izn0iz — CHANGES_REQUESTED** [review]
I think this broke thumbnail upload for videos

**Comment by sachin-into** [comment]
> I think this broke thumbnail upload for videos in web
here i have not changed video post creation just added UI for mobile app. i checked by commenting post creation function it did upload thumbnail image

**Review by omniwired — APPROVED** [review]

**Review by n0izn0iz — CHANGES_REQUESTED** [review]
music upload broken, it works on main but not on this branch
(pleas don't post test music on mainnet, use testnet for that, I can send you tokens if needed)

**Comment by sachin-into** [comment]
> music upload broken, it works on main but not on this branch (pleas don't post test music on mainnet, use testnet for that, I can send you tokens if needed)

I uploaded music on web from this branch and it worked.
https://app.teritori.com/feed/post/testori-91b040be-4fc5-4f08-b8e5-233216d51af1

Can you specify, what is the problem?

**Review by n0izn0iz — APPROVED** [review]
seems problem is on my side

**Pull request merged**